assets/src/blocks/Timeline/NewTimelineFrontend.js

```diff
@@ -61,28 +102,88 @@ export const NewTimelineFrontend = ({attributes}) => {
     })();
   }, [google_sheets_url]);
 
-  if (loading) {
-    return null;
+  // Format sheetData for frontend rendering
+  useEffect(() => {
+    document.body.classList.add('new-timeline-block'); // hack to load new styles for new timeline block
```

Comment:
This shouldn't be necessary 🤔 we should be able to load the styles via our `blocks.scss` file